index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div id="menu-section" class="menu-section">
+      <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+      <ul class="menu-list">
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="menu-item"><a href="#program">Program</a></li>
+          <li class="menu-item"><a href="#Join">Join</a></li>
+          <li class="menu-item"><a href="#sponsor">Sponsor</a></li>
+          <li class="menu-item"><a href="#news">News</a></li>
+      </ul>
+  </div>
+  <header class="header">
+    <i class="fa-brands fa-facebook"></i>
+    <i class="fa-brands fa-twitter"></i>
+    <a href="#" class="header-link">English</a>
+    <a href="#" class="header-link">My Page</a>
+    <a href="#" class="header-link">Logout</a>
+  </header>
+  <nav class="dnav">
+    <a href="#" class="navbrand">Web School</a>
+    <ul class="dnav-list">
+      <a href="./about.html" class="dnav-link"><li class="dnav-item">About</li></a>
+      <li class="dnav-item">program</li>
+      <li class="dnav-item">Join</li>
+      <li class="dnav-item">Sponsor</li>
+      <li class="dnav-item">News</li>
+      <li class="dnav-item"><button>School</button></li>
+    </ul>
+  </nav>
+  <section class="home">
+    <nav class="nav" id="menu-icon">
+      <a href="#menu" class="menu"><i class="fa-solid fa-bars"></i></a>
+      
+    </nav>
+    <div class="home-body">
+        <p class="body-disc">"Hello! Future Web Developers"</p>
+        <h1 class="home-heading">WE ARE HERE TO TRAIN THE BEST WEBSITE<span> DEVELOPERS FOR THE MARKET</span> IN 2023.</h1>
+        <p class="body-disc2">We provide the best suitable teaching method that can turn anyone to be the best in the market. And we also provide a flexable study time.</p>
+        <h2 class="heading2">Registration for 2023 is Open<span> (May16) ~ (May21)</span></h2>
+        <p class="home-footer">@Web school Ethiopia, Addis Ababa</p>
```

Comment:
- [ ] Kindly reduce your font sizes and use the same font-weight also color for the subheader. Also for the main big red title please increase the font weight and add a background as you see in the given design.
yours:
![image](https://user-images.githubusercontent.com/91486791/235130665-3ccce45a-7d9a-4b41-bf39-9de24dc330cc.png)
given:
![image](https://user-images.githubusercontent.com/91486791/235130821-4b360fad-07d0-43f6-83f8-284f435b5ecc.png)